src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -774,6 +781,17 @@ export class ActivityFeedItem extends React.Component<Props> {
         };
         break;
       }
+      case ARCHANOVA_WALLET_ENS_MIGRATION:
+        let statusLabel = isFailed ? this.STATUSES().FAILED : this.STATUSES().PENDING;
+        if (!isFailed && !isPending) statusLabel = this.STATUSES().COMPLETED;
```

Comment:
nit: This logic seems complex maybe we could nest ternary operator to make it easier to read.

```suggestion
        const statusLabel = isFailed 
          ? this.STATUSES().FAILED 
          : isPending 
            ? this.STATUSES().PENDING 
            : this.STATUSES().COMPLETED;
```

Reply:
I don't prefer nested ternary for me it's harder to read as it grows as well it's restricted by our lint rules, will keep as it is id you don't mind.